src/components/basic/Button/Button.mdx

```diff
@@ -0,0 +1,35 @@
+import { Story, Meta } from '@storybook/blocks'
+
+<Meta title="Design Guidelines/Buttons" />
+
+## Buttons
+
+### Buttons Pagination / Load More
+
+<img
+  width="739"
+  alt="image"
+  src="https://raw.githubusercontent.com/eclipse-tractusx/portal-assets/main/docs/static/button-load-more.png"
+/>
+
+<br />
+<br />
+
+Load More button component is based on the button component.
+
+##### Setting:
+
+- outlined
+- active
+- with icon
+
+<br />
+<br />
+
+## NOTICE
```

Comment:
@manojava-gk coming back to our discussion from yesterday, as you were asking me if an md file necessarily needs this notice footer (with this example https://github.com/catenax-ng/tx-portal-assets/blob/main/docs/user/Style-Guide/button%20load%20more.md), I was answering with yes because markdown files are considered documentation.
I realised after our discussion that your moving the contents into mdx files, which can be considered source code, so you should change to file header like for any other source file and remove the notice footer.